client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +144,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
```

Comment:
>I suppose we can duplicate storage in both Flux and Redux, if you think it'd be worthwhile.

A point to be made for implementing this in Flux is that we could detect whether the file being uploaded would push storage over the limit, which sounds as though it may currently be buggy anyways, but would likely be the intended behavior.

https://github.com/Automattic/wp-calypso/blob/73bc91f/client/lib/media/actions.js#L137-L139

(A challenge here is accessing the storage limit from within the Flux validation store)